test/built-ins/Promise/try/promise.js

```diff
@@ -0,0 +1,12 @@
+// Copyright (C) 2024 Jordan Harband. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+description: Promise.try return value is a Promise
+features: [promise-try]
+---*/
+
+var instance = Promise.try(function () {});
+
+assert.sameValue(instance.promise.constructor, Promise);
+assert.sameValue(instance.promise instanceof Promise, true);
```

Comment:
Because `Promise.try`'s return value is a `Promise`, `instance` should already be a `Promise`, therefore the `.promise` property access seems like a mistake
```suggestion
assert.sameValue(instance.constructor, Promise);
assert.sameValue(instance instanceof Promise, true);
```

Reply:
oh whoops, you're right, thanks